x/rewards/ante/fee_deduction.go

```diff
@@ -21,6 +22,11 @@ type BankKeeper interface {
 	BurnCoins(ctx sdk.Context, moduleName string, amt sdk.Coins) error
 }
 
+type CWGrantsKeeper interface {
```

Comment:
now that we have a module just for fees handling, would it make more sense to move the mPoG param and the fee deduction logic to that module? 🤔

Reply:
yes, let's open an issue!